test/suite/test_hs05.py

```diff
@@ -38,6 +37,7 @@
 class test_hs05(wttest.WiredTigerTestCase):
     # Force a small cache, but disable eviction of dirty pages until the cache is full.
     conn_config = 'cache_size=50MB,statistics=(fast),'
+    conn_config += 'eviction_trigger=100,'
```

Comment:
The intention here is likely to disable only dirty eviction hence the settings for dirty/updates target and trigger. But reading this test makes me wonder if we do not allow them to be equal, we cant set them to 100% each. What will that mean for "disabling" eviction in such cases.

Reply:
Correct, I will edit those settings as we don't allow them to be equal.
What I am thinking of doing is lowering `eviction_dirty_target` and `eviction_updates_target` to 95.
What do you think?

Reply:
I made the changes in 5d32ce3b05dd4e0ea6ed72f7647c5ef8efac3eaa